custom/enikshay/integrations/bets/const.py

```diff
@@ -1,3 +1,20 @@
+# BETS event names
+VOUCHER_EVENT_ID = "VOUCHER_EVENT_ID"  # TODO: Get this from BETS
+TREATMENT_180_EVENT = "104"
+DRUG_REFILL_EVENT = "109"
+SUCCESSFUL_TREATMENT_EVENT = "SUCCESSFUL_TREATMENT_EVENT"  # TODO: Get this from BETS
+DIAGNOSIS_AND_NOTIFICATION_EVENT = "103"
+AYUSH_REFERRAL_EVENT = "AYUSH_REFERRAL_EVENT"  # TODO: Get this from BETS
```

Comment:
Sounds good thanks Ethan. Should `chemist_voucher`/`lab_voucher` branch on the type of the host case? Or perhaps the `voucher_type` case property? Are the options for this value "prescription" and "test"?